mods/ra/maps/allies-08a/allies08a.lua

```diff
@@ -74,18 +74,6 @@ CreateScientists = function()
 	end)
 end
 
-FinishTimer = function()
```

Comment:
Why are we removing this instead of using it?

Reply:
Since the mission ends within seconds of the timer finishing, there is no need for the text to alert the player by flashing or for the text to disappear later. I honestly didn't know how this function worked when I copy/pasted it a couple years ago. 🤦

Reply:
Ok, that explains why we remove it. ^^